src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      this.result /= num;
+    },
```

Comment:
CODE STYLE ISSUE: You're missing a return statement here. For the sake of consistency, all your methods should return `this` to allow chaining. Please, add `return this;` at the end of the `divide` method.